frontend/src/api/files.ts

```diff
@@ -1,18 +1,23 @@
 import axios from "axios";
-import { FileHistoryCommit } from "models/FileHistoryCommit"
-import { TypeFileCommitEvolution } from "enum/TypeFileCommitEvolution"
+import { FileHistoryCommit } from "models/FileHistoryCommit";
+import { TypeFileCommitEvolution } from "enum/TypeFileCommitEvolution";
+import { FileData } from "models/FileData";
 
-export async function fileHistoryByDate(repositoryId: number): Promise<FileHistoryCommit[]> {
+export async function fileHistoryByDate(
+  repositoryId: number,
+  startDate: string,
+  endDate: string
```

Comment:
You added these arguments, but they are not used anywhere. 

You can pass these as query arguments to the api. Here's some example:
```
GET /repositories/123/files/stats/change-history?start_date=2024-10-14
GET /repositories/123/files/stats/change-history?start_date=2020-01-01&end_date=2024-01-01
GET /repositories/123/files/stats/change-history?end_date=2021-01-01
```

The result you will get from these api calls will be filtered by the start / end date.

Reply:
It's done